product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_x86_64/index.mdx

```diff
@@ -48,20 +44,14 @@ Operating system-specific install instructions are described in the correspondin
 
 - [AlmaLinux 8](jdbc_other_linux_8)
 
-- [CentOS 7](jdbc_centos_7)
-
 ### SUSE Linux Enterprise (SLES)
 
 - [SLES 15](jdbc_sles_15)
 
-- [SLES 12](jdbc_sles_12)
-
 ### Debian and derivatives
 
 - [Ubuntu 22.04](jdbc_ubuntu_22)
 
 - [Ubuntu 20.04](jdbc_ubuntu_20)
 
 - [Debian 11](jdbc_debian_11)
-
-- [Debian 10](jdbc_debian_10)
```

Comment:
Remove all the deprecated platform topics from navigation as well

Reply:
done